docs/usage/noise-reduction.md

```diff
@@ -107,7 +107,7 @@ Or perhaps at least weekly:
     {
       "matchPackageNames": ["/eslint/"],
       "groupName": "eslint",
-      "schedule": ["before 4am on monday"]
+      "schedule": ["* 0-3 * * 1"]
```

Comment:
```suggestion
      "schedule": ["* 0-4 * * 1"]
```

Reply:
before 4am but if we use range 0-4 then 4.15am will be allowed